uiautomationutilities/build.gradle

```diff
@@ -181,14 +181,8 @@ dependencies {
     implementation "androidx.test.uiautomator:uiautomator:$rootProject.ext.uiAutomatorVersion"
     implementation "androidx.test:monitor:$rootProject.ext.androidxTestMonitorVersion"
     implementation "androidx.test:core:$rootProject.ext.androidxTestCoreVersion"
-    if (findProject(":testutils") != null) {
-        //Use project reference if available
-        implementation(project(":testutils")) {
-            transitive = true
-        }
-    } else {
-        //Always use the latest release
-        implementation(group: 'com.microsoft.identity', name: 'testutils', version: '0.0.+')
+    implementation(project(":LabApiUtilities")) {
+        exclude module: 'appcompat'
```

Comment:
I don't think LabApiUtilities even has `appcompat` as `LabApiUtilities` doesn't have any Android dependencies. I think we can remove this `exclude` from here (and other similar places as well)